scripts/qa_crowbarsetup.sh

```diff
@@ -1105,7 +1105,7 @@ function onadmin_add_cloud_repo()
 function do_set_repos_skip_checks()
 {
     # We don't use the proper pool/updates repos when using a devel build
-    if iscloudver 6plus && [[ $cloudsource =~ develcloud ]]; then
+    if iscloudver 6plus && [[ $cloudsource =~ (M1|M2|M3|M4|M5|M6) ]]; then
```

Comment:
Should probably be: `if iscloudver 6 && [[ $cloudsource =~ ^M[1-6]+$ ]]``

The 6plus is definitely wrong, the other change is just consistency with what is already in the code.